eventDataScript.js

```diff
@@ -0,0 +1,388 @@
+//credit to w3schools for js tutorials, stackoverflow
+
+function loadPage()
```

Comment:
I'll call the loadpage method from the $(window).ready block

Also for the second comment about overwriting the team number but not the name, I had that as a feature in case someone messed up and didn't want to go through the process of deleting the team and then adding it. I basically said if the team exists but the number is different, override the number but keep the name and vice versa, but if nothing matches, then add it as a new team. Conversely, if everything matches, then say 'already stored'. I'll look more into it though.

I do need to revise the deleteTeam (eventData) and deleteElement (seasonConfig) methods, it seems that when I have nothing in the box, everything just disappears, like you were mentioning.

Reply:
You should make overwriting only happen on exact team number match. (like, it's a good idea, but there's a lot of team numbers that are similar, and I think that's what confused me).

Reply:
Ok, makes sense. It'll only overwrite the team number if the team number input matches.

Reply:
also, don't error on duplicate team names for different numbers - it is possible. Only numbers are guaranteed to be unique.

Reply:
ok, I think you have a mix up on editing teams:

![image](https://user-images.githubusercontent.com/2565361/68258240-bfb9a680-fffb-11e9-9054-975edee3255d.png)

I got into this state by trying to change team 123 from named "test" to "test1", this yields showing only "test1" in the list (dropping test2), and the change up of team numbers vs names in the cookies makes me think that you have an issue of assignment location.

Reply:
You should maybe consider using a dictionary, 

```js
Teams = {
123: "test1",
456: "test2",
}

set_cookie('/event_data teams/' + currCheckedSeason + '/teams', JSON.stringify(Teams));

raw_stored_data = get_cookie(('/event_data teams/' + currCheckedSeason + '/teams');
teams = JSON.parse(raw_stored_data);
teams[123] // test1
```

then, updating OR adding a team is 
```js
Teams[team_number] = team_name;
set_cookie('/event_data teams/' + currCheckedSeason + '/teams', JSON.stringify(Teams));
```

Reply:
Thank you! I didn't think of that but the dictionary works perfectly in this case as it doesn't allow for duplicate team numbers or keys.